src/main/scala/scalismo/common/interpolation/ValueInterpolator.scala

```diff
@@ -51,41 +51,44 @@ trait Interpolator[@specialized(Double, Float) A] {
     var mix: A = first
     var n = 1
     for (next <- rest) {
-      mix = blend(mix, next, n.toFloat / (n + 1))
+      mix = blend(mix, next, n / (n + 1))
```

Comment:
The conversion can not be ommited here. I'm quite sure it will end up as hinterher division, which is not what we want here.